src/domains/citizen-auth-app/01_network_itn.tf

```diff
@@ -0,0 +1,63 @@
+## itn-common vnet
+data "azurerm_resource_group" "italy_north_common_rg" {
+  name = format("%s-itn-common-rg-01", local.product)
+}
+
+data "azurerm_virtual_network" "common_vnet_italy_north" {
+  name                = format("%s-itn-common-vnet-01", local.product)
+  resource_group_name = data.azurerm_resource_group.italy_north_common_rg.name
+}
+
+data "azurerm_subnet" "pep" {
```

Comment:
```suggestion
data "azurerm_subnet" "itn_pep" {
```

Reply:
Done in [113b00f](https://github.com/pagopa/io-infra/pull/1037/commits/113b00f62a516e32aa40881b9fb625262ce49428)